common/scala/src/main/scala/org/apache/openwhisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -132,6 +132,10 @@ class KafkaConsumerConnector(
         } else {
           throw e
         }
+      case e: WakeupException =>
+        logging.info(this, s"WakeupException happened when do commit action for topic ${topic}")
+        recreateConsumer()
+        throw e
```

Comment:
If we recreate the consumer do we need to throw the exception?

Reply:
hm. have no need to throw the exception, because we already recreate the consumer.